api/lib/notifications/notificationController.js

```diff
@@ -26,17 +30,28 @@ class NotificationController {
 
   listenToNotifications() {
     eventSubscription((notification) => {
-      this.onNotification(notification)
+      this.onNotificationSendEMails(notification)
+      this.onNotificationSendPushNotifications(notification)
```

Comment:
here the same. Why are you using a plural? It is one push notification per notification logically

Reply:
well one per person

Reply:
same than above

Reply:
same than above XD

Reply:
:facepalm: